RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 1.35.0
 ------
 * Creating undo levels less frequently.
+* Reflect changes of slider in block settings immediately.
```

Comment:
Shouldn't this be [here](https://github.com/WordPress/gutenberg/blob/master/packages/react-native-editor/CHANGELOG.md) instead?